src/Wellcome.Dds/Wellcome.Dds.Dashboard/Controllers/WorkflowCallController.cs

```diff
@@ -156,5 +161,67 @@ public async Task<IActionResult> ResetMatchingErrors(string resetWithMessage)
             }
             return RedirectToAction("Errors");
         }
+
+        public async Task<IActionResult> Bulk(IFormFile identifiersFile)
+        {
+            var model = new BulkWorkflowModel();
+            if (identifiersFile is { Length: > 0 })
+            {
+                using (var reader = new StreamReader(identifiersFile.OpenReadStream()))
+                {
+                    model.Identifiers = await reader.ReadToEndAsync();
+                }
+
+                model.TidyIdentifiers();
+            }
+
+            return View(model);
+        }
+
+        public IActionResult BulkAnalyse(string identifiers)
+        {
+            var model = new BulkWorkflowModel
+            {
+                Identifiers = identifiers
+            };
+            model.TidyIdentifiers(true);
+            if (model.DdsIdentifiers.HasItems())
+            {
+                return View(model);
+            }
+
+            return View("Bulk", model);
+
+        }
+
+        public async Task<IActionResult> BulkWorkflow(BulkWorkflowModel model)
+        {
+            if (model.RunnerOptions.HasWorkToDo())
+            {
+                var options = model.RunnerOptions.ToInt32();
+                model.TidyIdentifiers(true);
+                model.WorkflowJobs = new List<WorkflowJob>();
+                try
+                {
+                    foreach (var ddsIdentifier in model.DdsIdentifiers)
+                    {
+                        var workflowJob = await workflowCallRepository.CreateWorkflowJob(
+                            ddsIdentifier.PackageIdentifier, options);
+                        model.WorkflowJobs.Add(workflowJob);
+                    }
+                }
+                catch (Exception e)
+                {
+                    model.Error = e.Message;
+                    logger.LogError(e, $"Could not create a workflow job, managed {model.WorkflowJobs.Count} out of {model.DdsIdentifiers.Count}.");
```

Comment:
Make compile time constant for structured logging. 
`logger.LogError(e, "Could not create a workflow job, managed {WorkflowJobsCount} out of {DdsIdentifiersCount}", model.WorkflowJobs.Count, model.DdsIdentifiers.Count);`